resolver-dns/src/main/java/io/netty/resolver/dns/DnsServerAddressStreamProvider.java

```diff
@@ -25,12 +25,29 @@
  * /etc/resolver</a>.
  */
 @UnstableApi
-public interface DnsServerAddressStreamProvider {
+public abstract class DnsServerAddressStreamProvider {
+    // TODO(scott): how is this done on Windows? This may require a JNI call to GetNetworkParams
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)